src/core/core.scale.js

```diff
@@ -782,13 +758,13 @@ module.exports = Element.extend({
 				if (position === 'top') {
 					y1 = alignPixel(chart, chartArea.top, axisWidth) + axisWidth / 2;
 					y2 = chartArea.bottom;
-					textBaseline = !isRotated ? 'bottom' : 'middle';
+					textOffset = ((!isRotated ? 0.5 : 1) - labelCount) * tickFont.lineHeight;
```

Comment:
Would it also make sense to capture `tickFont.lineHeight` into a variable to help minification (when used 3 or more times)? 
I think it would save a byte here :)

Reply:
Properties can't be minified, so the comparison is:

```javascript
// 38 bytes
a.lineHeight
a.lineHeight
a.lineHeight
```

vs

```javascript
// 20 bytes
,c=a.lineHeight
c
c
c
```